### PR TITLE
Add GIV token to ethereum prices

### PR DESCRIPTION
**Pull request by aminlatifi**
## Thank you for contributing to Spellbook 🪄

Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]
Added Giveth (giveth.io) GIV token to the list of ethereum tokens.
---

### CODE REVIEW

File: models/prices/ethereum/prices_ethereum_tokens.sql [hunk 1]
```diff
@@ -1790,7 +1790,8 @@ FROM
     ('gods-gods-unchained', 'ethereum', 'GODS', 0xccc8cb5229b0ac8069c51fd58367fd1e622afd97, 18),
     ('cah-moon-tropica', 'ethereum', 'CAH', 0x8e0e57dcb1ce8d9091df38ec1bfc3b224529754a, 18),
     ('bigtime-big-time', 'ethereum', 'BIGTIME', 0x64Bc2cA1Be492bE7185FAA2c8835d9b824c8a194, 18),
-    ('portal2-portal-coin', 'ethereum', 'PORTAL', 0x1bbe973bef3a977fc51cbed703e8ffdefe001fed, 18)
+    ('portal2-portal-coin', 'ethereum', 'PORTAL', 0x1bbe973bef3a977fc51cbed703e8ffdefe001fed, 18),
+    ('giv-giveth', 'ethereum', 'GIV', 0x900db999074d9277c5da2a43f252d74366230da0, 18)
```
Comment (jeff-dude) [inline models/prices/ethereum/prices_ethereum_tokens.sql:1794]: it appears this id value isn't valid. did you find the token on coinpaprika, grab the unique id on that page? can you double check and fix?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/prices/ethereum/prices_ethereum_tokens.sql (hunk 1 above)
Comment (aminlatifi) [reply models/prices/ethereum/prices_ethereum_tokens.sql:1794]: Hey @jeff-dude The token is not listed yet, I just up the token address!
Sorry for the confusion, we have submitted the request and will update the PR when it's ready. Our token is a listed in LBank exchange and it probably will be listed there.

Comment (aminlatifi) [reply models/prices/ethereum/prices_ethereum_tokens.sql]: @jeff-dude GIV is listed and I updated the id
https://coinpaprika.com/coin/giv-giv/

**Comment by aminlatifi** [comment]
@jeff-dude Can you rereview it? I think the problem is resolved.

File: models/prices/ethereum/prices_ethereum_tokens.sql (hunk 1 above)
Comment (jeff-dude) [reply models/prices/ethereum/prices_ethereum_tokens.sql]: nice -- looks good now!

**Comment by jeff-dude** [comment]
> @jeff-dude Can you rereview it? I think the problem is resolved.

yep, token is good to go now. we just need you to sign the CLA in order to get it merged. please follow instructions from bot comment above. you just need to simply copy/paste that command in a comment on it's own

**Comment by aminlatifi** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by aminlatifi** [comment]
recheck

**Comment by aminlatifi** [comment]
> > @jeff-dude Can you rereview it? I think the problem is resolved.
> 
> yep, token is good to go now. we just need you to sign the CLA in order to get it merged. please follow instructions from bot comment above. you just need to simply copy/paste that command in a comment on it's own

Done. Thanks @jeff-dude

**Comment by aminlatifi** [comment]
@jeff-dude The PR seems ready, I resolved the conflict with the latest version

**Review by jeff-dude — APPROVED** [review]
thank you!

**Pull request merged**